api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -299,34 +299,5 @@
 		<addForeignKeyConstraint constraintName="order_group_attribute_changed_by_fk" baseTableName="order_group_attribute" baseColumnNames="changed_by" referencedTableName="users" referencedColumnNames="user_id" />
 		<addForeignKeyConstraint constraintName="order_group_attribute_voided_by_fk" baseTableName="order_group_attribute" baseColumnNames="voided_by" referencedTableName="users" referencedColumnNames="user_id" />
 	</changeSet>
-
-	<changeSet id="2020-08-29-2200-TRUNK-5821" author="miirochristopher">
-		<preConditions onFail="MARK_RAN" onFailMessage="Table orders does not exist or the index orders_order_number already exists.">
-			<tableExists tableName="orders"/>
-			<and>
-				<not>
-					<indexExists indexName="orders_order_number" />
-				</not>
-			</and>
-		</preConditions>
-		<comment>Adding index to order_number column in Orders table</comment>
-		<createIndex  indexName="orders_order_number" tableName="orders">
```

Comment:
Only one liquibase-update-to-latest-version.xml file is run.
Therefore, removing the changesets from this file means that they will never be run on the current master branch.

Reply:
Thanks @dkayiwa !  So the correct thing to do would be to have these changesets in both update-to-latest-2.4.x and update-to-latest-2.3.x?

Reply:
Tru dat! 👍

Reply:
Thanks!  I updated this, merging now...